remoteappmanager/file_config.py

```diff
@@ -77,18 +70,18 @@ def __init__(self, *args, **kwargs):
             self.docker_host = "unix://var/run/docker.sock"
 
         self.tls_verify = (env.get("DOCKER_TLS_VERIFY", "") != "")
```

Comment:
DOCKER_TLS_VERIFY is not a 1=true 0=false . It's a "defined" vs "not defined" flag.

https://github.com/docker/docker/issues/22411